java/ql/lib/semmle/code/java/Expr.qll

```diff
@@ -1531,6 +1546,9 @@ class SwitchExpr extends Expr, StmtParent, @switchexpr {
     exists(YieldStmt yield | yield.getTarget() = this and result = yield.getValue())
   }
 
+  /** Holds if this switch has a case handling a null literal. */
+  predicate hasNullCase() { this.getAConstCase().getValue(_) instanceof NullLiteral }
```

Comment:
`ConstCase`s currently explicitly excludes `case null, default`, so this predicate seems incorrect (same thing goes for the mirrored predicate in `SwitchStmt`).